.studio/00_logging.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
This code is a combination of what used to be in .studio/logging and studio-common.  The 00_ file naming convention helps make sure this is loaded first so everything else can depend on the logging functions without re-sourcing them.